src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -278,6 +278,33 @@ const ScenarioParameters = ({
 
   const noTabsShown = parametersGroupsMetadata.length === 0;
   const isCurrentScenarioRunning = currentScenario.data.state === SCENARIO_RUN_STATE.RUNNING;
+  const isCurrentScenarioRejected = currentScenario.data.validationStatus === SCENARIO_VALIDATION_STATUS.REJECTED;
+  const isCurrentScenarioValidated = currentScenario.data.validationStatus === SCENARIO_VALIDATION_STATUS.VALIDATED;
+  const isEditDisabled =
+    noTabsShown || isCurrentScenarioRunning || isCurrentScenarioRejected || isCurrentScenarioValidated;
+
+  let disabledEditTooltip;
+  if (isCurrentScenarioRunning) {
```

Comment:
We can't use a `switch` statement here because the conditions are testing different variables (and not different values of a single variable) :smile: